examples/model_compress/slim_torch_cifar10.py

```diff
@@ -99,8 +108,12 @@ def main():
     print('=' * 10 + 'Test the pruned model before fine tune' + '=' * 10)
     pruner = SlimPruner(model, configure_list)
     model = pruner.compress()
-    test(model, device, test_loader)
-    # top1 = 93.55%
+    if args.parallel:
```

Comment:
No need to add args.parallel parameter,   this can be controlled by CUDA_VISIBLE_DEVICES if multiple GPUs on system.

Reply:
Personally I think it's okay to use an explicit "parallel" parameter in example code. Maybe some customers are not familiar with `CUDA_VISIBLE_DEVICES`.

Reply:
I agree with @liuzhe-lz .